docs/_includes/intro.html

```diff
@@ -22,25 +22,36 @@
             </div>
         </div> 
         <div class="forkme-banner-text">   
-                 <div >
-                    <h1>Getting Started Samples</h1>           
-                    <h2 class="nolineheading">{{ site.title }}</h2>    
-                        <!-- Introduce the samples with some text. Below is an example. -->
-                        <p>These tutorials will get you to up to speed in sending messages with Solace technology. There are two ways you can get started:</p>
-                        <ol >
-                            <li><strong class="">If your company has Solace message routers deployed</strong>, contact your middleware team to obtain the following:
-                                <ul>
-                                    <li>A host name or IP address of a Solace router</li>
-                                    <li>A username and password to access the router</li>
-                                    <li>A VPN in which you can produce and consume messages</li>
-                                </ul>
-                            </li>
-                            <li><strong>If you do not have access to a Solace message router</strong>, see "<a href="{{ site.docs-vmr-setup }}" target="_top" style="text-decoration:none; border-style: none;    "  onmouseover="this.style.textDecoration='none'; this.style.color='#F36F23';" 
-    onmouseout="this.style.textDecoration='none';this.style.color='#005dab';" >Set up a VMR</a>" to download and install the software.</li>
-                        </ol> 
-                </div>
+            <h1>Learn about the {{ site.title }}</h1>
+
+            <div class="intro">
+                <p>Solace Getting Started tutorials get you to up to speed and quickly sending messages with Solace technology.</p>
```

Comment:
This sentence is awkward. Using both speed and quickly so close together is redundant. Try:

Solace Getting Started tutorials get you up to speed on how to send and receive messages with Solace technology.